lexer.go

```diff
@@ -97,9 +103,39 @@ func (s *sshLexer) lexRvalue() sshLexStateFn {
 			s.skip()
 			return s.lexVoid
 		case '#':
+			if !isSingleQuoted && !isDoubleQuoted {
```

Comment:
I think this is no longer needed.

Reply:
I don't quite understand the logic of openssh. When is a comment and when isn’t it? I can't tell yet whether this modification is appropriate.

Reply:
+1, I cannot describe clear behaviour of openssh. It looks like a duck type. I decide to remove handle of single quote and double quote currently. I'll reopen another pr after  I have a deep learning of openssh.